spinta/types/model.py

```diff
@@ -263,6 +265,9 @@ def _link_prop_enum(
         for enums_ in enums:
             if enums_ and prop.given.enum in enums_:
                 return enums_[prop.given.enum]
+        if isinstance(prop.dtype, Money):
+            if len(prop.given.enum) == 3:
+                return
```

Comment:
This code should not go here. Since enums depends on dtype, then linking enums, should be moved out of `Property` in to `Integer` and `String` datatypes, where enums can be applied.

Reply:
Is this code should be moved in dtype link function?

https://github.com/atviriduomenys/spinta/blob/8c9be652efcf99e197f17d3e2472de9397a0e8c4/spinta/types/money/link.py

Reply:
Yes.